rasa/cli/data.py

```diff
@@ -234,6 +260,72 @@ def split_nlu_data(args: argparse.Namespace) -> None:
     telemetry.track_data_split(args.training_fraction, "nlu")
 
 
+def augment_nlu_data(args: argparse.Namespace) -> None:
```

Comment:
I think we should move this function outside of the `cli` module? Look like this is the function which is actually performing the augmentation strategy @federicotdn

Reply:
`augment_nlu_data(...)` is currently only reading in all input files, determining the intents and creating the training data and paraphrase pools and then running the two augmentation strategies. Do you suggest moving the intent and pool generation business out of the `cli` module?

Reply:
Yes, I think the logic of what the command is doing internally shouldn't be in the `cli` module but inside a function outside of it. So in this case the whole implementation logic should sit inside the module which actually performs the augmentation. Look at how `cli/train.py` implements some of the train commands.